docs/main_viewer.md

```diff
@@ -0,0 +1,127 @@
+# Main Viewer
+
+![](./pngs/blank_main_viewer.png)
+
+When openend, AMP will display its main image viewer.  This window will contain figures generated
+by plugins as well as direct images loaded in via the 'Open Cohort' menu option.
+
+### Cohort Loading
+
+To load a cohort of images into the viewer, find the `Open Cohort...` option under the `File`
+section of the upper taskbar.
+
+![](./pngs/open_cohort.png)
+
+This will open a file explorer.  Selecting and opening a folder in this explorer will have `AMP`
+walk through the subdirectories within said folder until it finds the most shallow depth containing
+`.tif(f)` files.  It will then load the directory structure from the selected folder, through said 
+depth, into the `Files` section of the main viewer.
+
+For example, for selecting the folder `slide01` with structure:
+
+```
+slide01
+|
+|------- fov1
+|          |-------- TIFs
+|                     |-------- Au.tif
+|                     |-------- betacat.tif
+|                     ...
+|
+|------- fov2
+|          |-------- TIFs
+|                     |-------- Au.tif
+|                     ...
+...
+```
+
+will find all of the FOV's channels.  As a counter-example, selecting the folder `bad_slide01`:
+
+```
+bad_slide01
+|
+|------- fov1
+|          |-------- TIFs
+|                     |-------- Au.tif
+|                     |-------- betacat.tif
+|                     ...
+|
+|------- fov2
+|          |-------- TIFs
+|                     |-------- Au.tif
+|                     ...
+|
+|------- bad.tif
+...
+```
+
+will only load in tifs at the same file depth as `bad.tif`.
+
+<div 
+    style="
+        border: 0px solid #35f;
+        border-left-width: 6px;
+        padding: 8px;
+        margin-bottom: 8px;
+        background-color: #acf;
+        color: #000"
+> 
+    <b><i>Note</i></b>:  The file explorer can only search for tifs at file depths less than 6,
+    relative to the selected folder.
+</div>
+
+![](./pngs/cohort_file_explorer.png)
```

Comment:
Now that you have such a nice explanation of the directory structure for loading, I think this screenshot is not needed/confusing. It doesn't have the file structure you specified before, so it's not clear what it's illustrating